packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -140,7 +140,32 @@ export default class EmailCardDropRequestsRoute {
       return;
     }
 
-    let verificationCode = (crypto.randomInt(1000000) + '').padStart(6, '0');
+    let unclaimedButExisting = await this.emailCardDropRequestQueries.query({
+      emailHash: normalizedEmailHash,
+      ownerAddress: ctx.state.userAddress,
+    });
+
+    if (unclaimedButExisting.length > 0) {
+      let existingRequest = unclaimedButExisting[0];
+      let updatedEmailCardDropRequest = await this.emailCardDropRequestQueries.updateVerificationCode(
+        existingRequest.id,
+        generateVerificationCode()
```

Comment:
Why do we need to update the verification code?

Reply:
hmm… I guess I’m just following the convention, I’m not sure why the convention exists though 🤔